FAQ.md

```diff
@@ -1,5 +1,58 @@
 # Frequently Asked Questions
 
-## ???
+## How to use `log` or `tracing` as default log backend
 
-???
+ * **log**, add the feature flag `logger_log` to the dependency in `Cargo.toml`
+
+  ```toml
+  elkodon = { version = "0.1.0", features = ["logger_log"]}
+  ```
+
+ * **tracing**, add the feature flag `logger_tracing` to the dependency in `Cargo.toml`
+
+ ```toml
+  elkodon = { version = "0.1.0", features = ["logger_tracing"]}
+ ```
+
+## A crash leads to the failure `PublishSubscribeOpenError(UnableToOpenDynamicServiceInformation)`
+
+**Note:** A command line tool and internal service is already planned to cleanup
+resources from crashed applications, see issue #65.
+
+When an application crashes some resources may remain in the system and must be
+cleaned up manually. If this occurs, stop all services and remove manually all
+shared memory segments and static service config files.
+
+```sh
+rm -rf /dev/shm/*
```

Comment:
This is dangerous since other applications might also use shared memory and have created files. Is there already an elkodon prefix/postfix?

Reply:
I introduced a global prefix that is used for every file elkodon creates. It can be configured in `config.toml`.
With this, I adjusted the recommendation to `rm -rf /dev/shm/elk_*`